frontend/src/employee-frontend/components/unit/tab-groups/groups/Group.tsx

```diff
@@ -216,6 +224,16 @@ export default React.memo(function Group({
     !unit.type.includes('CLUB') &&
     requireRole(roles, 'ADMIN', 'UNIT_SUPERVISOR')
 
+  const showChildOccupancyFactor =
+    !unit.type.includes('CLUB') &&
+    requireRole(
+      roles,
+      'ADMIN',
+      'UNIT_SUPERVISOR',
+      'SPECIAL_EDUCATION_TEACHER',
+      'SERVICE_WORKER'
+    )
```

Comment:
You shouldn't use these role checks in front end. I think the simplest check would be to check that the data exists.